jbpm-test-coverage/src/test/java/org/jbpm/test/functional/jobexec/LogCleanupCommandTest.java

```diff
@@ -217,6 +217,28 @@ public void deleteAllLogsOlderThanTomorrowDifferentProcess() throws Exception {
         Assertions.assertThat(getNodeInstanceLogSize(HELLO_WORLD_ID)).isPositive();
     }
 
+    @Test(timeout = 10000)
+    public void deleteAllLogsPage() throws InterruptedException {
```

Comment:
I believe now this exception is no longer thrown
```suggestion
    public void deleteAllLogsPage() {
```